config-model/src/test/java/com/yahoo/vespa/model/content/RedundancyTest.java

```diff
@@ -26,4 +27,19 @@ private static Redundancy createRedundancy(int redundancy, int implicitGroups, i
         return r;
     }
 
+    private static void verifyFinalRedundancy(Redundancy redundancy, int expectedFinal, int expectedEffectiveFinal) {
+        assertEquals(expectedEffectiveFinal, redundancy.effectiveFinalRedundancy());
+        assertEquals(expectedFinal, redundancy.finalRedundancy());
+        assertEquals(expectedEffectiveFinal, redundancy.effectiveReadyCopies());
+        assertEquals(expectedFinal, redundancy.readyCopies());
+    }
+
+    @Test
+    void test_that_redundancy_is_rounded_up() {
+        verifyFinalRedundancy(new Redundancy(1, 1, 1, 5, 5), 1,5);
+        verifyFinalRedundancy(new Redundancy(1, 1, 1, 5, 4), 1,4);
+        verifyFinalRedundancy(new Redundancy(1, 2, 2, 5, 10), 2,10);
+        verifyFinalRedundancy(new Redundancy(1, 2, 2, 5, 9), 2,9);
```

Comment:
Nit: missing spaces (unless this is intentional for visual grouping)

Reply:
Thank, Unintentional, not so easy to see when Intellij clutters the text with parameter names.